example/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   cke:
     container_name: cke
-    image: quay.io/cybozu/cke:1.23.0
+    image: quay.io/cybozu/cke:1.23.5
```

Comment:
Will this tag be updated after release?

Reply:
Yes, Update here after the image is pushed.